apple/Sources/FerrostarMapLibreUI/Views/DynamicallyOrientingNavigationView.swift

```diff
@@ -48,7 +50,10 @@ public struct DynamicallyOrientingNavigationView: View, CustomizableNavigatingIn
         navigationState: NavigationState?,
         minimumSafeAreaInsets: EdgeInsets = EdgeInsets(top: 16, leading: 16, bottom: 16, trailing: 16),
         onTapExit: (() -> Void)? = nil,
-        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] }
+        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] },
+        @ViewBuilder currentRoadNameViewBuilder: @escaping (String?) -> AnyView = { name in
+            AnyView(CurrentRoadNameView(currentRoadName: name, theme: DefaultRoadNameViewTheme()))
+        }
```

Comment:
In an effort to keep these views clean. Maybe best as a view modifier when not using the default behavior? Thinking something using the ideology of [SpeedLimitViewModifier](https://github.com/stadiamaps/ferrostar/pull/287/files#diff-51fe331771b082e102612f3578d010c8dd830dc07b3738524cb1759be11722f7). But more like 
```swift
func navigationCurrentRoadView(@ViewBuilder currentRoadNameViewBuilder: @escaping (String?) -> AnyView)
```